### ISSUE TITLE
[wishlist] Adding addtional actions in app/scope info

### ISSUE THREAD

**Issue by cibersheep** [bhdouglass/falcon#2]
It would be nice (specially as a replacement for actual apps scope) in addition to Favirite and Launch to have:
 - Hide. To hide an app or scope from main Falcon scope
 - Info. To show info from the store and a method to rate
 - Remove. To uninstall app/scope from device

**Comment by bhdouglass** [comment]
Thanks for the suggestions! Hiding apps would be easy, the hard part would be showing them again. I could probably pull some info from uApp Explorer in the app preview.

Being able to rate apps and uninstall them would be difficult as there would be a bunch of hoops to jump through in order to do it. It would be easier to just link to the click store scope, but I can't directly link to an app (known limitation). I think a better option would be to have a dedicated app store app.

**Comment by cibersheep** [comment]
I know it's a lot of wishes :)

I was thinking in how to be able to make apps visible again... Something that could work nicely it would be, marking the apps as invisible and in options adding the possibility to hide them in the same way of "Show apps id...". I mean, an toggle.
What do you think?

**Comment by bhdouglass** [comment]
Yeah that's kinda what I was thinking, the scope settings are just a little limited so we'll see how it goes.

**Comment by bhdouglass** [comment]
I was checking out the libertine scope today and it has a rather clever way of hiding/showing apps. You can see the hidden ones by going into the search section and tapping on the "hidden apps" department. So I think I'll probably do something like that.

**Comment by cibersheep** [comment]
That sounds a way more elegant indeed

**Comment by Telstan** [comment]
Is there a way to hide the native Ubuntu app tray?  I love your Falcon Launcher and the carousel option sets it apart from Android and IOS. It is fret to have the native one popping up when I scroll.

**Comment by bhdouglass** [comment]
@Telstan unfortunately I don't think you can remove the integrated apps scope, but you could stick it at the end of your scope list.
